cumulusci/tests/pytest_plugins/pytest_sf_orgconnect.py

```diff
@@ -113,7 +114,12 @@ def org_config(request, current_org_shape, cli_org_config, fallback_org_config):
     # fast running test suites it might return a hardcoded
     # org and for integration test suites it might return
     # a specific default org or throw an exception.
-    return org_config
+    with mock.patch.object(OrgConfig, "latest_api_version", "48.0"), mock.patch.object(
+        OrgConfig, "refresh_oauth_token"
+    ):
+        print("XXX yieldiing orgconfig", OrgConfig.refresh_oauth_token)
```

Comment:
should we log and not print this to maintain CumulusCI's use of logger?

Reply:
Fixed in 3ba5ceedafe which I guess I forgot to push.